exercises/build.gradle

```diff
@@ -12,18 +12,14 @@ subprojects {
 
   apply plugin: "checkstyle"
 
-  // Add a task that copies test source files into a build directory. All @Ignore annotations
-  // are removed during this copying, so that when we run the copied tests, none are skipped and
-  // all should execute.
-  //
-  // Note that journey-test.sh also strips @Ignore annotations using sed. The
-  // stripping implementations here and in journey-test.sh should be kept
-  // consistent.
+  // Add a task that copies test source files into a build directory.
+  // All @Ignore and @Disabled annotations are removed during this copying,
+  // so that when we run the copied tests, none are skipped and all should execute.
```

Comment:
This change might warrant a separate PR, as it changes something relatively important about the tests.

Reply:
Yes you’re right, thanks for pointing it out. I guess it should have been included in #2354 since that is the first time we’ve used JUnit 5, but I guess I overlooked it.

Will open a separate PR for this later today, since it’s necessary to correctly test exercises with JUnit 5 in the CI pipeline.

Reply:
Opened #2363 to address this. I'll revert the changes here and we'll wait for that PR to be merged first before merging this PR.